turbopack/crates/turbopack-trace-server/src/reader/mod.rs

```diff
@@ -249,19 +314,23 @@ impl TraceReader {
     fn wait_for_more_data(
         &mut self,
         file: &mut TraceFile,
-        initial_read: &mut Option<u64>,
-        format: Option<&dyn TraceFormat>,
+        initial_read: &mut Option<(u64, Instant)>,
+        format: Option<&ErasedTraceFormat>,
     ) -> Option<bool> {
         let Ok(pos) = file.stream_position() else {
             return Some(true);
         };
-        if let Some(total) = initial_read.take() {
+        if let Some((total, start)) = initial_read.take() {
             if let Some(format) = format {
                 let stats = format.stats();
                 println!("{}", stats);
             }
             if total > MIN_INITIAL_REPORT_SIZE {
-                println!("Initial read completed ({} MB)", total / (1024 * 1024));
+                println!(
+                    "Initial read completed ({} MB, {}s)",
+                    total / (1024 * 1024),
+                    (start.elapsed().as_millis() / 100) as f32 / 10.0
+                );
```

Comment:
https://doc.rust-lang.org/std/fmt/#precision

```suggestion
                println!(
                    "Initial read completed ({} MB, {:.1}s)",
                    total / (1024 * 1024),
                    start.elapsed().as_millis() as f32 / 1000.0
                );
```